esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3153,8 +3176,35 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
         if (!validNameFilter)
             throw MakeStringException(ECLWATCH_INVALID_INPUT, "Invalid Name Filter '*'");
 
+        double version = context.getClientVersion();
+        bool serverFound = false;
+        unsigned filesFound = 0;
+        IArrayOf<IEspPhysicalFileStruct> files;
 #ifdef _CONTAINERIZED
-        UNIMPLEMENTED_X("CONTAINERIZED(CFileSprayEx::onDropZoneFileSearch)");
+        bool isIPAddressReq = isIPAddress(dropZoneServerReq);
```

Comment:
Could this code be used for bare-metal and containerized?

Reply:
Yes, I will change it